Block/Product/YotpoReviewRenderer.php

```diff
@@ -18,12 +18,26 @@ public function getReviewsSummaryHtml(
         $displayIfNoReviews = false
     ) {
         
+        $objectManager = \Magento\Framework\App\ObjectManager::getInstance();
+        $storeManager = $objectManager->get('\Magento\Store\Model\StoreManagerInterface');
+        $cp = $objectManager->get('\Magento\Framework\App\Request\Http');
         $enableBottomlineCategoryPage = $this->isBottomlineCategoryEnabled();
-	$enableMagentoDefaultReviews = $this->isMagentoDefaultReviewsEnabled();
+		$enableMagentoDefaultReviews = $this->isMagentoDefaultReviewsEnabled();
+		
         if ($enableBottomlineCategoryPage) {
-            return $this->showCategoryBottomLine($product);
+			if ($cp->getFullActionName()==='cms_index_index'){
+				return $this->showCategoryBottomLine($product);
```

Comment:
note that you can move $cp->getFullActionName() to variable an use it in the next if statements for example:
$actionName = $cp->getFullActionName()

Reply:
Hi Dan,
Hope you are doing well. 
I had changed and deployed at UAT as well.
Regards,
Ijaz Ali